hazelcast/src/main/resources/hazelcast-config-5.2.xsd

```diff
@@ -3313,7 +3313,8 @@
 
     <xs:complexType name="native-memory">
         <xs:all>
-            <xs:element name="size" type="memory-size" minOccurs="0"/>
+            <xs:element name="size" type="capacity" minOccurs="0"/>
+            <xs:element name="capacity" type="capacity" minOccurs="0"/>
```

Comment:
What if there are both parameters?

Reply:
Last one will be picked. TBH, if you have any other idea how to keep XSD backward compatible and up-to-date with Java config, I'd be super grateful, I don't like this repetition as well :P

Reply:
I am fine with last one wins.